internal/testdb/testdb.go

```diff
@@ -0,0 +1,48 @@
+package testdb
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/smartcontractkit/chainlink/v2/core/cmd"
+	"github.com/smartcontractkit/chainlink/v2/core/store/dialects"
+)
+
+// TODO rename?
+func DropAndCreateThrowaway(parsed url.URL, postfix string, empty bool) (string, error) {
```

Comment:
```suggestion
func CreateEphemeralDb(parsed url.URL, postfix string, empty bool) (string, error) {
```

Is there a sensible path forward to separating the concerns of dropping the db and creating a new one? 

Or is the right thing to leave 'postgres' alone and use a new name each time?

Reply:
Maybe a hook to close the db when the test completes?

Reply:
We already have a cleanup routine elsewhere, but IIUC this is just a simple way to remain robust through failures :shrug:

Reply:
Updated to `CreateOrReplace` to align with SQL.